app/screens/home/search/search.tsx

```diff
@@ -89,11 +94,16 @@ const SearchScreen = ({teamId}: Props) => {
 
     const {scrollPaddingTop, scrollRef, scrollValue, onScroll, headerHeight, hideHeader} = useCollapsibleHeader<FlatList>(true, onSnap);
 
+    const handleTextChange = useCallback((newValue: string) => {
+        setSearchValue(newValue);
+        setCursorPosition(newValue.length);
+    }, [setSearchValue, setCursorPosition]);
```

Comment:
`setX` functions (when they come from a `useState` in this component) do not need to be in the dependencies. (`useState` takes care of them being constant between renders)